webclient/pid/model_serializers.py

```diff
@@ -165,6 +166,10 @@ def _serialize_subjects(self):
     def _serialize_types(self):
         pass
 
+    @abstractmethod
```

Comment:
Asi je zbytečné to vytvářet jako `@abstractmethod`, když je to implementované pouze u 1 ze 3 potomků. Tím pádem to bude stačit přidat k lokalitě. Plus bych možná zvážil, jestli tam místo `pass` nedat `return None` nebo `return []`.

Reply:
No a ale to volání je už v tom obecném modelu (třída serialize_publish), tak by to tu být mělo, ne? A definice je v dokumentu, ne v lokalitě. To pass opravím.